shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/instance/ComputeNodeInstance.java

```diff
@@ -32,6 +32,8 @@
     
     private InstanceDefinition instanceDefinition;
     
+    private String xaRecoveryId;
```

Comment:
Where to set XaRecoveryId? It seems always null

Reply:
`xaRecoveryId` is used for pre-occupancy, used to store instance-level attributes for narayana, initialization is supplemented by @menghaoranss